generated_code/aderdg.py

```diff
@@ -180,6 +178,14 @@ def extractTractions(self):
   def mapToTractions(self):
     return self.extractTractions().T
 
+  def setBottomToIdentity(self, selectVelocitySpp):
```

Comment:
Would be better `setVelocitiesToIdentity`; but cf. comment above: you can take `extractVelocities` and transpose instead.